src/Collections/Sequences/Sorting.dfy

```diff
@@ -0,0 +1,191 @@
+// RUN: %dafny /compile:0 "%s"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT 
+*******************************************************************************/
+
+include "../../StandardLibrary.dfy"
+include "../../UInt.dfy"
+include "../Sequences/Seq.dfy"
+
+module Sorting {
+  export
+    reveals Reflexive, AntiSymmetric, Connected, TotalOrdering
+    reveals LexicographicByteSeqBelow, LexicographicByteSeqBelowAux
+    provides AboutLexicographicByteSeqBelow
+    provides SortedBy, MergeSortBy, MergeSortedBy
+    provides StandardLibrary, UInt
+
+  import StandardLibrary
+  import opened UInt = StandardLibrary.UInt
+  import opened Seq
+
+  predicate Reflexive<T(!new)>(R: (T, T) -> bool) 
```

Comment:
Could you put Reflexive, AntiSymmetric, Transitive, etc. in a file named `Relations.dfy` instead of "standardLibrary"? Here this is too generic for "Sorting", and this entire project is supposed to be the standard library, so there is no point having a file named "standard library", its name should more reflect what it defines.